src/main.cpp

```diff
@@ -1,101 +1,140 @@
 #include <util/JvmException.hpp>
+
```

Comment:
It is just a matter of preference: separating the `#include <...>` from the `#include "..."`. For me it organizes the code if only just a little. Also, I just realized that "util/JvmException.hpp" is actually inside the project, so it would be better to use `#include "..."` instead. But I leave to you, the better way...

Reply:
Feel free to modify to "..."